tests/setup/odh-core.yaml

```diff
@@ -83,7 +84,17 @@ spec:
         name: manifests
         path: data-science-pipelines-operator/
     name: data-science-pipelines-operator
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: codeflare-stack
+    name: codeflare-stack
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: ray/operator
+    name: ray-operator
   repos:
   - name: manifests
-    uri: https://github.com/opendatahub-io/odh-manifests/tarball/master
+    uri: https://github.com/anishasthana/odh-manifests/tarball/add_distributed_workloads_component
```

Comment:
@anishasthana does this need to get changed back to `uri: https://github.com/opendatahub-io/odh-manifests/tarball/master` ?

Reply:
Good catch on this!

Reply:
Fixed